src/home/NotificationBox.tsx

```diff
@@ -98,10 +98,11 @@ export function useSimpleActions() {
   }, [])
 
   const superchargeRewards = useSelector((state) => state.supercharge.availableRewards)
+  const cloudBackupCompleted = useSelector(cloudBackupCompletedSelector)
 
   const actions: SimpleAction[] = []
   if (!backupCompleted) {
-    if (showKeylessBackup) {
+    if (showKeylessBackup && !cloudBackupCompleted) {
```

Comment:
should the cloud backup check be in the if above? We don't want to show any backup related home card if they restored right? Maybe worth doing a migration to rename `backupCompleted` or make a wrapper selector that checks both

Reply:
Yea you're right, i should move it up